src/components/Signals/SignalsProvider.tsx

```diff
@@ -26,33 +24,28 @@ export const useSignalContext = () => {
 type SignalCallback = (data: BuzzUpdateSignalSchema | any) => void;
```

Comment:
I'm guessing this is no longer used for buzz only, so it'd be best to defer type definition when declaring the callback
```suggestion
type SignalCallback = (data: unknown) => void;
```